POS/Operation/03_EditBillList.md

```diff
@@ -3,181 +3,172 @@
-    The detail view of the product is displayed in the bill list.
+1. Click the offer in the bill list on which you want to apply a discount.   
+    The detail view of the offer is displayed in the bill list.
 
-    ![Product Detail](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/ProductDetails.png "[Product Detail]")
+    ![Offer detail](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/ProductDetails.png "[Offer detail]")
 
-2. Click on the drop-down list to the right of the *Discount* field and select the **%** option if you want to apply a percentage discount or select the **EUR** option to apply a fixed amount discount to the product.   
+2. Click the drop-down list to the right of the *Discount* field and select the **%** option if you want to apply a percentage discount or select the **EUR** option to apply a fixed amount discount to the offer.   
     The selected option is displayed in the drop-down list.
 
     > [Info] The currency displayed in the drop-down list corresponds to the default currency in the tab *Global Settings*.
 
 3. Enter a number in the text field to the right of the *Discount* field. The number indicates the amount of the discount, depending on the selection in the drop-down list as a percentage or as a fixed amount.
 
-4. Click the [SAVE] button in the upper right corner to apply the discount.   
-    The detail view of the product is closed. The bill list is displayed again. The reduced price of the product is displayed in red, the discount is indicated below the price.
+4. Click the [SAVE] button in the bottom right corner to apply the discount.   
+    The detail view of the offer is closed. The bill list is displayed again. The reduced price of the offer is displayed in red. The discount is displayed below the price.
 
-    > [Info] You can apply an individual discount to each product in your bill list. The discount is applied to the product entry in the bill list. That means, if the quantity of a product in the bill list is greater than 1, the discount will apply to all products of this entry. If you want to apply a discount to a single one of several products in an entry, you have to use the [single scan function](./02_SelectOffer.md#use-single-scan) first to add the product as a single entry in the bill list.
+    ![Offer discount](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/ProductDiscountBillList.png "[Offer discount]")
 
 
-### Add a discount to the purchase
+## Edit the price of an offer
+
+You can manually change the price of an offer in your bill list, for example, if a price is incorrect in the system.   
+If you enter a changed price for an offer, you will not be able to apply an additional single discount to that offer. The discount field will be locked.  
+The modified price is related to each item of the offer entry in the bill list. That means, if the quantity of an offer in the bill list is greater than one, the modified price is applied to each item of this entry.  
```

Comment:
if the quantity of an offer is greater than one -> s. Kommentar oben

Reply:
das sollte man verstehen